{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -288,7 +288,7 @@ class MediaRootS3Boto3Storage(S3Boto3Storage):
 
 sentry_logging = LoggingIntegration(
     level=SENTRY_LOG_LEVEL,  # Capture info and above as breadcrumbs
-    event_level=None,  # Send no events from log messages
+    event_level=logging.ERROR  # Send errors as events
```

Comment:
Build is failing the Black code formatting due to the missing comma. I wish the output was better, but this should fix it:

```suggestion
    event_level=logging.ERROR,  # Send errors as events
```

Reply:
Thank you for the correction. 

For future reference, how did you spot the error? Going to Travis shows many failures all around. Searching for the code (line 291) I couldn't find it in the report.

Reply:
I simply knew how Black wanted this to be formatted by experience. The output is pretty bad for this in Travis, if you know way to improve it, help is welcome.